conduit/pipeline/pipeline_test.go

```diff
@@ -178,15 +178,15 @@ func (m *mockImporter) OnComplete(input data.BlockData) error {
 	if m.onCompleteError {
 		err = fmt.Errorf("on complete")
 	}
-	m.finalRound = input.BlockHeader.Round
```

Comment:
I'm fine with committing things like this for now as incremental improvements. But eventually the `data.BlockData` input won't be using go-algorand types and so doing `input.BlockHeader.Round` would just return a `types.Round` anyways.